.github/workflows/ci.yml

```diff
@@ -112,9 +112,27 @@ jobs:
     steps:
       - run: true
 
+  publish-check:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release'
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: ${{ env.RUST_VERSION }}
+
+      - name: Publish crate check
+        uses: katyo/publish-crates@v2
```

Comment:
Does `uses` actually run the action? I'm not super familiar with actions but does this need a `run:`

Reply:
Yep, you can see it works for [`fuel-core`](https://github.com/FuelLabs/fuel-core/actions/runs/4027955287/jobs/6924289655)